app/lib/search/README.md

```diff
@@ -12,27 +12,16 @@ We collect the following scores prior to building the search index:
 The health score should reflect the general code health of the package,
 and it is calculated for each package version separately.
 
-The health score is calculated from the `pana` analysis results, via
-`AnalysisView.health`. The score is normalized to the `[0.0 .. 1.0]`
-range using the package's `fitness` values from `pana`, which includes
-the *magnitude* of the package (approximates code size and complexity),
-and its *shortcomings* (a combination of single-line issues and errors
-which are measured in proportion to the *magnitude*). 
+Details can be found on the following page:
+https://pub.dartlang.org/help#health
```

Comment:
This is a bit self-referential. Can you also describe where the logic is in the source code, or where the scoring description on the help page is in the source code?